Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -223,6 +232,32 @@ public class PBXProjGenerator {
             return Set(files.map { $0.buildFile.reference })
         }
 
+        func getRunScript(runScript: RunScript) throws -> PBXShellScriptBuildPhase {
+
+            let shellScript: String
+            switch runScript.script {
+            case let .path(path):
+                shellScript = try getPath(path).read()
+            case let .script(script):
+                shellScript = script
+            }
+            let escapedScript = shellScript.replacingOccurrences(of: "\"", with: "\\\"").replacingOccurrences(of: "\n", with: "\\n")
```

Comment:
You're right, this should go into PXBWriter, but I just put it here for now to get it working